codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaders.java

```diff
@@ -41,8 +41,22 @@
 /**
  * Provides the constants for the standard HTTP header names and values and
  * commonly used utility methods that accesses an {@link HttpMessage}.
+ * <p>
+ * Concrete instances of this class are most easily obtained from its default factory: {@link #DEFAULT_HEADER_FACTORY}.
  */
 public abstract class HttpHeaders implements Iterable<Map.Entry<String, String>> {
+    /**
+     * Default implementation of {@link HttpHeadersFactory}, that creates an {@link HttpHeaders} instance that has the
+     * recommended header validation enabled.
+     */
+    public static final HttpHeadersBuilder DEFAULT_HEADER_FACTORY = HttpHeadersBuilder.DEFAULT;
```

Comment:
On the same note, mismatch between type name and constant name (builder vs factory) makes it confusing. 

Consider aligning the name and instead of adding new public constants on `HttpHeaders`, either make them public on `HttpHeadersBuilder` (that way it's easier to discover) or consider going with static methods.